src/scripts/h5p-multi-media-choice-option.js

```diff
@@ -249,6 +250,27 @@ export class MultiMediaChoiceOption {
     this.wrapper.appendChild(this.accessibilitySolutionText);
   }
 
+  /**
+   * Shows feedback below answer (if any is available)
+   */
+  showFeedback() {
+    if (!this.feedback) {
+      return;
+    }
```

Comment:
Matter of taste: blank lines between coherent code blocks can make reading code easier.